src/static/images/cart.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
let's discuss static assets folder + we can think of creating a component for svg's